playbooks/byo/openshift-cluster/config.yml

```diff
@@ -3,6 +3,19 @@
   tags:
   - always
 
+- name: Verify Requirements
+  hosts: OSEv3
```

Comment:
It would be best to use the right groups in the checks (the way checks are structured, they all run under one task which runs against all hosts, so they filter themselves out by various criteria). It's just not obvious to me what the groups would ideally be, since I haven't grokked what the various groups are for and where they come from.